lib/graphiti/util/serializer_attributes.rb

```diff
@@ -106,7 +106,7 @@ def wrap_proc(inner)
         typecast_ref = typecast(Graphiti::Types[@attr[:type]][:read])
         ->(serializer_instance = nil) {
           val = serializer_instance.instance_eval(&inner)
-          if Graphiti.config.typecast_reads
+          if Graphiti.config.typecast_reads && inner.nil?
```

Comment:
@klobuczek this was added after our last discussion,
The `inner` represent here that the attribute has defined a custom rendering block in resource class.
So in such case the value to be rendered is retuned in the custom rendering block, then it shouldn't get modified here by typecasting. 

e.g for created_by attribute for `UserResource` 
```
class UserResource
  attribute created_by, :datetime do
    @object.strftime('%d%m%y-%h%m')
  end
end
```
for rendering of created_by attribute the typecasting was overriding the format in which date time was being rendered